src/parser.ts

```diff
@@ -167,17 +175,20 @@ function handleToken(
   const isLeafNode = typeName === 'image';
   if (!isLeafNode) {
     for (const child of token.children)
-      handleToken(child, nodes, file, inlineParent);
+      handleToken(child, nodes, file, handleSlots, inlineParent);
   }
 
   nodes.pop();
 }
 
-export default function parser(tokens: Token[], file?: string) {
+export default function parser(tokens: Token[], args?: string | ParserArgs) {
```

Comment:
This is a breaking change huh? We should denote that so we make sure to bump the version appropriately.

Reply:
I was trying to avoid making this a breaking change, which is why I made it accept a string (like the existing signature) or an object. Am I missing something?

Reply:
Ohhhhhh. I see. Missed that.